### DISCUSSION TITLE
PEP735 claim about PDM

### DISCUSSION THREAD

**Discussion by baggiponte** [pdm-project/pdm#2575] [General]
Hey there, I was reading the PEP 735 proposal and read [this](https://pep-previews--3541.org.readthedocs.build/pep-0735/#object-specification-does-not-allow-for-pep-508-decomposition):

> Poetry and PDM both allow for their object formats for dependency data to be used to decompose a PEP 508 specification into parts. For example, a dependency on requests==2.21.0 could be written under these tools in a form like {name = "requests", version = "==2.21.0"}.

I don't think PDM supports this, am I right? If so, I could point it out in the PEP discussion.

**Comment by frostming** [comment]
Yes, PDM doesn't support object formats
